lib/resourceapply/machineconfig_test.go

```diff
@@ -285,7 +286,7 @@ func TestApplyMachineConfig(t *testing.T) {
 	for idx, test := range tests {
 		t.Run(fmt.Sprintf("test#%d", idx), func(t *testing.T) {
 			client := fake.NewSimpleClientset(test.existing...)
-			_, actualModified, err := ApplyMachineConfig(client.MachineconfigurationV1(), test.input)
+			_, actualModified, err := ApplyMachineConfig(context.TODO(), client.MachineconfigurationV1(), test.input)
```

Comment:
https://pkg.go.dev/context#Background suggests that tests should use `context.Background` — unless you plan to actually exercise the context cancellation in a future test enhancement.

(Applies throughout.)

Reply:
Oh, getting tests would be even better, of course.